config/settings/base.py

```diff
@@ -77,30 +81,59 @@
 X_FRAME_OPTIONS = "DENY"  # Header: X-Frame-Options DENY
 
 # Content Security Policy
+# Use CSP_STYLE_SRC_ELEM / exclude inline styles once browser support is better
 CSP_STYLE_SRC = (
     "'self'",
-    "'unsafe-inline'",
```

Comment:
Do we have to remove this? I find inline CSS to be much more readable and maintainable for one-off changes.